PerimeterXModule/DataContracts/Cookies/Base/BaseDecodedCookie.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System.Runtime.Serialization;
+
+namespace PerimeterX
+{
+    public abstract class BaseDecodedCookie
+    {
+
+        [DataMember(Name = "u")]
+        public string Uuid { get; set; }
+        [DataMember(Name = "v")]
+        public string Vid { get; set; }
+        [DataMember(Name = "t")]
+        public double Time { get; set; }
+
+        public abstract string GetBlockAction();
+        public abstract double GetScore();
+        public abstract bool IsCookieFormatValid();
+
+        public string GetUUID() {
```

Comment:
I thought it would be good to force all cookies to return the values that we know it should return